### ISSUE TITLE
[Bug] Last Modified or logged in user update not logging/tracking in Salesforce inspector

### ISSUE THREAD

**Issue by ambikaarora2304** [tprouvot/Salesforce-Inspector-reloaded#465]
**Make sure to read the troubleshooting section before creating an issue**

- [ ] I've read the [common issues](https://tprouvot.github.io/Salesforce-Inspector-reloaded/troubleshooting/) and the one I'm about to create is not one of those documented.

**Describe the bug**
Considering logged in as User A and loginAs User B, if we try to perform an update from the Salesforce UI the lastmodified by is User B, but if we are using Salesforce Inspector reloaded to update the data (on show all data page for example) the last modified by User A instead of User B.

**To Reproduce**
1. Go to 'Account Record'
2. Click on 'Salesforce inspector reloaded', login in as user B
3. Try to update account field eg - account status from the salesforce inspector, show all data
4. The account status will be updated with User A instead of User B

**Expected behavior**
It should be updated by User B as we have logged in with User B

**Desktop (please complete the following information):**
 - OS: [e.g.MacOS]
 - Browser [e.g. chrome]
 - Version [e.g. v1.23]

**Additional context**

**Comment by tprouvot** [comment]
Note: this behaviour occurs only when a connected app is used

**Comment by tprouvot** [comment]
Hi @ambikaarora2304,
One question : is the user B allowed to use API based on your conencted app configuration ?

**Comment by ambikaarora2304** [comment]
Yes, user B is allowed to use connected app as that's something we are managed via permission set

**Comment by ambikaarora2304** [comment]
Hi, Any updates on the fix here? or any ETA which you can share with us?

**Comment by tprouvot** [comment]
Hi @ambikaarora2304 ,
No ETA on this at the moment I did not had the time to search deeper.
